themes/digital.gov/layouts/partials/core/collection.html

```diff
@@ -90,9 +90,7 @@ <h2>{{- .heading -}}</h2>
   {{- $src := .src -}}
   {{- with .item -}}
     <div class="icon">
-      <a href="{{- $href -}}" title="{{- .Title | markdownify -}}">
-        <img src="{{ $src | relURL }}" alt="{{- .Params.name }} icon"/>
-      </a>
+      <img src="{{ $src | relURL }}" alt="{{- .Params.name }} icon" aria-hidden="true"/>
```

Comment:
We can remove `alt` description since it's being `aria-hidden`.

On USWDS we've been including empty alt attributes with `aria-hidden` for strictly decorate images.

Reply:
Description is now removed.